plugins/woocommerce/includes/data-stores/abstract-wc-order-data-store-cpt.php

```diff
@@ -482,15 +482,15 @@ public function update_order_from_object( $order ) {
 			array(
 				'ID'                 => $order->get_id(),
 				// phpcs:ignore WordPress.DateTime.RestrictedFunctions.date_date -- use of date is intentional.
-				'post_date'          => date( 'Y-m-d H:i:s', $order->get_date_created( 'edit' )->getOffsetTimestamp() ),
+				'post_date'          => gmdate( 'Y-m-d H:i:s', $order->get_date_created( 'edit' )->getOffsetTimestamp() ),
 				'post_date_gmt'      => gmdate( 'Y-m-d H:i:s', $order->get_date_created( 'edit' )->getTimestamp() ),
 				'post_status'        => $this->get_post_status( $order ),
 				'post_parent'        => $order->get_parent_id(),
 				'post_excerpt'       => method_exists( $order, 'get_customer_note' ) ? $order->get_customer_note() : '',
 				'post_type'          => $order->get_type(),
 				// phpcs:ignore WordPress.DateTime.RestrictedFunctions.date_date -- use of date is intentional.
-				'order_modified'     => ! is_null( $order->get_date_modified() ) ? date( 'Y-m-d H:i:s', $order->get_date_modified( 'edit' )->getTimestamp() ) : '',
-				'order_modified_gmt' => ! is_null( $order->get_date_modified() ) ? gmdate( 'Y-m-d H:i:s', $order->get_date_modified( 'edit' )->getTimestamp() ) : '',
+				'order_modified'     => ! is_null( $order->get_date_modified() ) ? gmdate( 'Y-m-d H:i:s', $order->get_date_modified( 'edit' )->getOffsetTimestamp() ) : '',
+				'order_modified_gmt' => ! is_null( $order->get_date_modified() ) ? gmdate( 'Y-m-d H:i:s', $order->get_date_modified( 'edit' )->getOffsetTimestamp() ) : '',
```

Comment:
```suggestion
				'order_modified_gmt' => ! is_null( $order->get_date_modified() ) ? gmdate( 'Y-m-d H:i:s', $order->get_date_modified( 'edit' )->getTimestamp() ) : '',
```

Reply:
Should this be analogous to `post_date` and `post_date_gmt`?

Reply:
This should be similar now, let me know if you meant something else. We use `order_modified_gmt` so that we can filter it in `update_post_modified_data` method (hooked on `wp_insert_post_data`).